cni-plugin/integration/manifests/calico/calico-install.yaml

```diff
@@ -0,0 +1,3755 @@
+---
+# Source: calico/templates/calico-config.yaml
```

Comment:
TIOLI: Are all these manifests taken from a single URL, or is this file the result of concatenating multiple files? If the latter, it might be better to put them separately under a directory, mirroring the layout upstream, to more easily upgrade when the time comes...

Reply:
I did this part so I can explain: they are the output of `linkerd install-cni` with appropriate arguments.

Reply:
It's the former. Originally we applied it straight from the internet...but! Lossy networks, unreliable links, and complicated logic made me think it's better to just `curl` a copy locally. It makes it easier to maintain a directory for each scenario and apply it _all_ instead of maintaining separate logic branches that are based on the scenario being run.

I think in the long run your concern is well founded though, we'll need to find a way to upgrade or version calico. We should perhaps move to `helm` at some point. If we ever want to do more advanced testing (e.g ebpf mode, different other modes such as policy engines) we'll need a more expressive way of installing anyway.

Reply:
Cool, this should do for now then :+1: